### ISSUE TITLE
FixNum/Float/Boolean window names cause error

### ISSUE THREAD

**Issue by ncoop** [tmuxinator/tmuxinator#387]
This is very much like #313, but for window names.
tmuxinator 0.7.1
tmux 2.1
ruby 2.3.0

Sample yaml:
```
# ~/.tmuxinator/fixnum-test.yml
---
name: fixnum-test
root: ~/

windows:
  - 234:
```

Sample output:
```
$ mux start fixnum-test 
undefined method `shellescape' for 234:Fixnum
```

Chaining a `to_s` in `Window#initialize` seems to fix it:
```
      @name = if !window_yaml.keys.first.nil?
                window_yaml.keys.first.to_s.shellescape
              end
```

**Comment by ethagnawl** [comment]
@ncoop That fix looks reasonable. Do you want to submit a PR?

**Comment by ethagnawl** [comment]
Addressed in #388.
